Taula d'estadístics descriptius

sandraam22 opened the pull request:
He afegit una funció que crea un data.frame amb estadístics descriptius

sandraam22 committed:
Taula d'estadístics descriptius per una variable numèrica.
Primer verifica que la variable és numèrica i desprès calcula els estadístics.
Els valors s'emmagatzemen en un data frame

sandraam22 committed:
Merge pull request #5 from irene187/main

Taula d'estadístics descriptius

irene187 reviewed (changes requested):
bones sandra, que guai la proposta, m'agrada molt a més l'estructura del codi d'agrupar per separat noms i resultats! Estem aquí amb el manel i el miquel fent proves de què passa amb la opció de revisió de codi i request changes. aleshores, si ens tires un cable, et demanem un canvi, a veure què passa. Per exemple, reordenar els resultats de la taula aixi: mitjana, variancia, dt, mínim, Q1, Q2, Q3, màxim, N, NA. I potser si t'animes, afegir el coeficient de variació (100 * dt/mitjana). 
merci!

sandraam22 committed:
Update estadistics_descriptius.R

He canviat l'ordre dels estadístics i he afegit el coeficient de variació (100*dt/mitjana).

sandraam22 commented:
Hola Irene,
Gràcies!
D'acord, he fet els canvis que proposeu però he afegit també el valor "mínim" perquè potser us l'heu deixat en l'enumeració d'estadístics que m´heu proposat.
Per qualsevol cosa, en parlem!

sandraam22 commented:
Crec que he tancat el comentari i alhora el pull request... encara estic aprenent!

sandraam22 committed:
Corregim un error en el codi (hi havia un ")" de més

irene187 committed:
Merge branch 'main' into sandra

irene187 commented:
crec que ja ho he integrat. anem a mirar-ho :)
